pages/vi/profiles/Going-Gone.md

```diff
@@ -0,0 +1,28 @@
+# Daren Sell
+
+<img src="https://images-ext-2.discordapp.net/external/CEZuWLsRIiHvgEXmkzFszvzjmdiRrrzzrTMhkVXqbgA/https/i.gyazo.com/thumb/1200/8279b3bc404a283f3ac4100e5a785054-png.jpg"
```

Comment:
The purpose of having Markdown is to make pages easy to write and read. In general, we would like to avoid adding HTML tags in our Markdown files. Could you embed this image using Markdown?

Reply:
Yep i sure can! Thank you for the feedback.